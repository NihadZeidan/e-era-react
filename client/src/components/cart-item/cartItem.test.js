import { shallow } from "enzyme";
import CartItem from "./cartItem.component";

it("Should render cartDropDown Component", () => {
  const mockCartItem = {
    name: "test name",
    price: 5,
    imageUrl: "test url",
    quantity: 1,
  };

  const wrapper = shallow(<CartItem item={mockCartItem} />);
  expect.assertions(2);

  expect(wrapper).toMatchSnapshot();
  expect(wrapper.exists()).toEqual(true);

});
