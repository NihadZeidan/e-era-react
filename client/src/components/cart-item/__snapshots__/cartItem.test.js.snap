// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render cartDropDown Component 1`] = `ShallowWrapper {}`;
